docs/source/index.rst

```diff
@@ -0,0 +1,60 @@
+.. QuCoNot documentation master file, created by
+   sphinx-quickstart on Tue Nov 22 10:09:26 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to QuCoNot's documentation!
+===================================
+
+**QuCoNot** is a Python Library for all your MCT needs.
+
+Check out the :doc:`usage` section for further information, including how to :ref:`install <installation>` the project.
+
+.. note::
+
+   This project is under active development.
+
+It creates MCT circuits depending on user resource restrictions.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Contents:
+
+   implementations
+   usage
+   api
+   contribution
+
+Support
+=======
+If you are having issues, please let us know by filling an issue on our `issue tracker <https://github.com/QuCoNot/QuCoNot/issues>`_.
+
+Authors
+=======
+QuCoNot is the work of following people (in alphabetical order):
+
+* Shraddha Aangiras
```

Comment:
Better: write your three names, and then under supervision of me + Ozlem (this is more fair to you(